java/src/main/java/io/crate/jdbc/sample/DataProvider.java

```diff
@@ -48,6 +47,20 @@ public DataProvider() throws SQLException {
         }
     }
 
```

Comment:
I'd add an additional method `getProperty(String name)` to the `DataProvider` class and make `properties()` private.